features/admin/edit_a_service.feature

```diff
@@ -0,0 +1,33 @@
+@edit-a-service
+Feature: Edit a service
+
+Scenario: Only Service Manager can edit services
+  Given I am logged in as the production admin-ccs-category user
+  # TODO get me here from the search
```

Comment:
Same as with #424, I reckon a new admin step `I am on a random g-cloud service page` will do the job here.  Although then we can't so easily check the initial values without patching up something ugly like `I see '£that service.minPrice to £that service.maxPrice per that service.unit per that service.interval' text on the page` (I am NOT suggesting we do this!!!)